parsl/executors/base.py

```diff
@@ -103,7 +103,7 @@ def scaling_enabled(self) -> bool:
         pass
 
     @abstractmethod
-    def create_monitoring_info(self, status: Dict[object, JobStatus]) -> List[object]:
+    def create_monitoring_info(self, status: Dict[object, JobStatus], block_id_type: str) -> List[object]:
```

Comment:
this signature makes that parameter compulsory, which is probably the right thing to do.

But not every call to create_monitoring_info supplies a block_id_type, I think

Reply:
Added `block_id_type` to every call